code/__DEFINES/DNA.dm

```diff
@@ -126,6 +126,7 @@
 #define NOHUSK			22 // Can't be husked.
 #define ROBOTIC_LIMBS	23 //limbs start out as robotic; but also use organic icons. If you want to use the default ones, you'll have to use on_species_gain
 #define NOMOUTH			24
+#define SQUIDLEGS       25 // Like the lizard's digitigrade feet, but for squids. Currently is only relevant for wearing socks
```

Comment:
Rename this to NOSOCKS for future use on other species and to reduce bloat.

Reply:
Yeah. The main reason I named it 'SQUIDLEGS' was because I thought it might be useful for maintaining and fiddling with squids. Seeing that a lot of their design is focused around their legs, you could instead combine all of their unique leg-based features into a single trait, which could also reduce bloat and allow for the transfer of said traits onto other mobs without having to write new code. Though I admit I'm too stupid to do that and no one really *wants* to maintain squids, do they?

Reply:
Squids are honestly a few inches from getting dropped entirely.